src/passes/externalContractHandler/index.ts

```diff
@@ -0,0 +1,16 @@
+import { AST } from '../../ast/ast';
+import { ASTMapper } from '../../ast/mapper';
+
+import { ContractDefinition } from 'solc-typed-ast';
+
+import { ExternalContractInterfaceInserter } from './externalContractInterfaceInserter';
+
```

Comment:
I'd appreciate a comment here outlining the overall transform and more importantly why it's needed.